modules/cudaoptflow/perf/perf_optflow.cpp

```diff
@@ -326,56 +326,7 @@ PERF_TEST_P(ImagePair, OpticalFlowDual_TVL1,
     }
 }
 
-//////////////////////////////////////////////////////
-// NvidiaOpticalFlow_1_0
```

Comment:
`cv::cuda::NvidiaOpticalFlow_1_0` is not removed from OpenCV public API, it is just disabled (throws exceptions).

Please restore this test.

Reply:
Restored. Thank you